app/src/input_manager.c

```diff
@@ -394,6 +404,19 @@ input_manager_process_key(struct input_manager *im,
     bool shift = event->keysym.mod & KMOD_SHIFT;
     bool repeat = event->repeat;
 
+    static SDL_Keycode lastKeycode = -1;
+    static Uint16 lastMod = -1;
+    static int repeatCount = 0;
+    
+    if(down && keycode == lastKeycode && event->keysym.mod == lastMod){
```

Comment:
nit: in the code base, there is a space after `if` and before `{` (there are other missing spaces in the PR)

Reply:
I'll look into them. I may have missed something along the way. Thanks.

Reply:
I think I got them all.